content/documentation/overview/index.adoc

```diff
@@ -14,21 +14,22 @@ toc::[]
 :icons: font
 :imagesdir: /images/documentation/overview/
 
-== What is Kiali?.
+== What is Kiali?
 
-Kiali provides answers to the question: What are the microservices in my Istio service mesh doing ?
+Kiali provides answers to the questions: What are the microservices in my Istio service mesh, and how are they connected?
 
 [#img-homepage]
 .Kiali Graph
 image::kiali.png[Demo Website]
 
 {nbsp} +
+// Credits to Daniel Oh: http://superuser.openstack.org/articles/microservices-istio-service-mesh-doing/
```

Comment:
I'm a bit concerned here, did Daniel Oh was informed that we're using his text here?
I think it should be fair to mention it's a quote. Or rewrite.

Reply:
Actually, the real author was @jmazzitelli :-) https://github.com/kiali/kiali/commit/de6e744f217fd9529f619401bb0561e74a60c506